resident-services/README.md

```diff
@@ -0,0 +1,72 @@
+### Contains
+* This folder contains performance test script of Resident Service Endpoints.
+    1. Authorize​ Admin​ Validate Token 
+    2. Login​ Redirect URI 
+    3. VID Policy
+    4. Generate VID
+    5. Get VIDs of the Resident
+    6. Service History
+    7. Revoke VID
+    8. Auth Lock Unlock
+    9. Auth Lock Status 
+    10. Request Card VID 
+    11. Service History Download
+
+
+* Open source Tools used,
+    1. [Apache JMeter](https://jmeter.apache.org/)
+
+### How to run performance scripts using Apache JMeter tool
+* Download Apache JMeter from https://jmeter.apache.org/download_jmeter.cgi
+* Download scripts for the required module.
+* Start JMeter by running the jmeter.bat file for Windows or jmeter file for Unix. 
+* Validate the scripts for one user.
+* Execute a dry run for 10 min.
+* Execute performance run with various loads in order to achieve targeted NFR's.
+
+### Setup points for Execution
+
+* Create Identities in MOSIP Authentication System (Setup) : This thread contains the authorization api's for regproc and idrepo from which the auth token will be generated. There is set of 4 api's generate RID, generate UIN, add identity and add VID. From here we will get the VID's which can be further used as individual id for required Resident Service endpoints. These 4 api's are present in the loop controller where we can define the number of samples for creating identities in which "addIdentitySetup" is used as a variable.
```

Comment:
addIdentitySetup - I think we sould change this variable name, it is very confusing. We can change it as "freshIdentityCreationCount"

Reply:
update the reference in readme also

Reply:
Done